### ISSUE TITLE
Define IOP as a living specification

### ISSUE THREAD

**Issue by sandersaares** [Dash-Industry-Forum/DASH-IF-IOP#180]
I propose that we take example from [WHATWG HTML](https://html.spec.whatwg.org/) and make IOP a living specification aimed at promoting modern practices for creating and consuming interoperable DASH.

Components of this would be:

* Explicitly stating this in the document itself.
* Removal of a version number, just keeping a last update timestamp.

And I think that is enough in terms of actual changes of substance.

This would enable authors to easily answer with a strong YES questions like "Is it OK to remove dubious descriptor X?" (such as came up a few weeks ago with some audio point).

It would also encourage keeping a forward-thinking outlook and avoid getting bogged down in historical details (very important to encourage contributions, which are the lifeblood of IOP).

This is already de-facto the case IMO and would help align reader and author expectations.

**Comment by acbegen** [comment]
I'd support this approach. Should help us move from the dreaded word files as well.

**Comment by haudiobe** [comment]
I do not believe this is possible and I am not sure this is a good idea for someone referencing us. You need to create a certain amount of stability and not just arbitrarily change. And to @acbegen it is really important that we active contributors to the activities. We need to discuss this in the IOP if and how we flag this to IOP.
